webserver/static/gulpfile.js

```diff
@@ -113,13 +112,22 @@ function buildScripts() {
   ]).then(writeManifest);
 }
 
-gulp.task('styles', buildStyles);
+gulp.task('styles', function () {
+  return buildStyles(writeManifest);
+});
 gulp.task('scripts', buildScripts);
 
 gulp.task('watch', ['styles', 'scripts'], function () {
   let watch = require('gulp-watch');
 
-  watch(path.resolve(STATIC_DIR, '**/*.less'), buildStyles);
+  watch(path.resolve(STATIC_DIR, '**/*.less'), function () {
```

Comment:
I worked this out, it's because it's the `watch` method on `gulp`, not part of `gulp-watch`